src/catalog/proc_catalog.cpp

```diff
@@ -17,20 +17,23 @@
 #include "executor/logical_tile.h"
 #include "storage/data_table.h"
 #include "type/value_factory.h"
+#include "expression/expression_util.h"
+#include "codegen/buffering_consumer.h"
 
 namespace peloton {
 namespace catalog {
 
 #define PROC_CATALOG_NAME "pg_proc"
 
-ProcCatalogObject::ProcCatalogObject(executor::LogicalTile *tile,
+ProcCatalogObject::ProcCatalogObject(codegen::WrappedTuple wrapped_tuple,
```

Comment:
Same as above. Is a reference not good enough? This applies to all such uses in other constructors as well